assets/prototype/domain/shared/predicates/datetimes/capacityBelow.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { is } from 'ramda';
+
+/**
+ * Internal dependencies
+ */
+import { Datetime } from '../../../eventEditor/data/types';
+import validFiniteReglimit from './validFiniteReglimit';
+import validInfiniteReglimit from './validInfiniteReglimit';
```

Comment:
I think we should avoid using `RegLimit` and rather use `capacity` to be more clear about what it means. Capacity is also an attribute shared by other entities like `Venue`. So, we can reuse any common logic to format/process that capacity value for all of them.

Reply:
Replaced that with `capacity`, thx.